README.Rmd

```diff
@@ -21,14 +21,14 @@ knitr::opts_chunk$set(
 [![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
 <!-- badges: end -->
 
-The goal of repro is to make the setup of a reproducible workflow with RMarkdown, Git, Make & Docker as easy as possible. To that end it builds upon the great [`usethis`-package](https://github.com/r-lib/usethis). The reproducible workflow mirrors the one proposed by [Peikert & Brandmaier (2019)](https://psyarxiv.com/8xzqy/).
+The goal of `repro` is to make the setup of reproducible workflows as easy as possible. To that end it builds upon the great [`usethis`-package](https://github.com/r-lib/usethis). As `repro` curently undergoes major changes to support more workflows then just the one proposed by [Peikert & Brandmaier (2019)](https://psyarxiv.com/8xzqy/), you may want to take a look at [`worcs`](https://github.com/cjvanlissa/worcs).
```

Comment:
I would wait a week, and then say: You may want to install the worcs package on CRAN for a user-friendly workflow that meets most requirements of reproducibility and open science.

This is the benefit of getting worcs on CRAN now: We can start accumulating users before everything is totally perfect.

Reply:
I like it.